File: ANL/Lista1/zadanie3.java
public class zadanie3 {
    public static void main(String[] args) {
        for(double i=2;i<51;i++){
            System.out.println(f(i));
        }
    }
    public static double f(double x){
        if(x==0){
            return 1.0;
        }
        if(x==1){
            return (-1.0/7.0);
        }
        double f1=69.0*f(x-1);
        double f2=10.0*f(x-2);

        return (1.0/7.0)*(f1+f2);
    }
}

File: WWW/lista5/zadanie3.js
$(document).ready(function () {
    $.getJSON("http://localhost/zadanie3/zadanie3.json",function (data) {
        let template = "<ul> {{#res}}{{/res}} </ul>"
        let text= Mustache.render(template, data);
        $("#Lista").append(text)
    })

})